app/javascript/Pages/components/CardPopular.scss

```diff
@@ -0,0 +1,49 @@
+$primary-color: #6600cc;
+$dark-color: #252636;
+$white-color: #ffffff;
+$gray-color: #aaaaaa;
+
+.Media {
+  display: flex;
+  align-items: flex-start;
+  width: 21.5rem;
+  border-radius: 0.6rem;
+  overflow: hidden;
+  cursor: pointer;
+  margin-top: 1rem;
+  background-color: $dark-color;
+}
+
+.Media-figure {
+  left: 0;
+  top: 0;
+  width: 5.5rem;
+  height: 5rem;
+  border-radius: 0.313rem;
+}
+
+.Media-body {
+  flex: 1;
+  height: 5rem;
+  color: $white-color;
+  vertical-align: middle;
+
+  .Media-body-info {
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+    margin-left: 0.625rem;
+
+    .Media-teacher {
+      font-weight: 500;
+      font-size: 1rem;
+      line-height: 1.5rem;
+    }
+
+    .Media-skill {
```

Comment:
**issue (blocking)**: don't indent classes like this, the only elements you should indent are states (`:hover`, `.active`, ...) or situations where something depends on another (`.alternate-parent &`)